src/metabase/driver/sql/query_processor.clj

```diff
@@ -621,19 +621,22 @@
 
 (defn rewrite-fields-to-force-using-column-aliases
   "Rewrite `:field` clauses to force them to use the column alias regardless of where they appear."
-  [form]
+  [form & {is-breakout :is-breakout}]
```

Comment:
I would prefer a separate arity 2 variant, because that gives better chances for linters to catch mistakes.